yarn-project/circuits.js/src/structs/kernel/__snapshots__/index.test.ts.snap

```diff
@@ -13,6 +13,74 @@ public_inputs: [
 proof_witness_indices: [ 7 8 9 10 11 12 ]
 has_data: 0
 
+read_requests:
```

Comment:
I guess we should create a follow-up PR that the ordering circuit has different public inputs from inner kernels since the ordering circuit should not need to output any reads to rollup circuit.

Reply:
Followup ticket: https://github.com/AztecProtocol/aztec-packages/issues/1074